cpp/perspective/CMakeLists.txt

```diff
@@ -264,16 +267,31 @@ elseif(PSP_CPP_BUILD OR PSP_PYTHON_BUILD)
 	#####################
 	# VANILLA CPP BUILD #
 	#####################
-	if(CMAKE_BUILD_TYPE_LOWER STREQUAL debug)
-		set(OPT_FLAGS " \
-			-O0 \
-			-g3 \
-			")
+	if(WIN32)
+		if(CMAKE_BUILD_TYPE_LOWER STREQUAL debug)
+			set(OPT_FLAGS " \
```

Comment:
@sc1f we should set  debug flags (see prev comment) instead of having this opt flags thing